nf_core/lint.py

```diff
@@ -451,6 +486,25 @@ def check_nextflow_config(self):
             else:
                 self.passed.append((4, "Config variable process.container looks correct: '{}'".format(container_name)))
 
+    def check_actions_branch_protection(self):
+        """Checks that the github actions workflows are valid.
+
+        Makes sure that ``nf-core lint``runs in github actions workflows and that 
+        tests run with the required nextflow version.
+        """
+        for wf in ['.github/workflows/branch.yml']:
+            fn = os.path.join(self.path, wf)
+            if os.path.isfile(fn):
```

Comment:
I don't think that we need to check this as the tests will have already failed if this file doesn't exist.

Reply:
so far they just produce a warning, though

Reply:
Ah good point..